cdap-docs/developers-manual/source/getting-started/standalone/docker.rst

```diff
@@ -98,43 +117,101 @@ started correctly.
    with the Docker VM's IP address (such as ``192.168.99.100``) that you obtained earlier.
    Start a browser and enter the address to access the CDAP UI from outside Docker.
 
-#. To control the CDAP instance, use this command, substituting one of ``start``, ``restart``, ``status``,
-   or ``stop`` for ``<command>``:
+Options Starting CDAP Containers
+--------------------------------
 
-   .. tabbed-parsed-literal::
-     :tabs: "Linux or Mac OS X",Windows
-     :mapping: linux,windows
-     :dependent: linux-windows
-     :languages: console,shell-session
-     
-     .. Linux or Mac OS X
-     
-     $ docker exec -d cdap-standalone /opt/cdap/sdk/bin/cdap sdk <command>
+- Starting the Standalone CDAP, in the background (default execution)::
 
-     .. Windows
-     
-     > docker exec -d cdap-standalone /opt/cdap/sdk/bin/cdap sdk <command>
+    $ docker run -d --name cdap-sdk caskdata/cdap-standalone
 
-#. When you are finished, stop CDAP and then shutdown Docker:
+- Use the CDAP CLI within the above *cdap-standalone* container::
 
-   .. tabbed-parsed-literal::
-     :tabs: "Linux or Mac OS X",Windows
-     :mapping: linux,windows
-     :dependent: linux-windows
-     :languages: console,shell-session
-     
-     .. Linux or Mac OS X
-     
-     $ docker exec -d cdap-standalone /opt/cdap/sdk/bin/cdap sdk stop
-     $ docker-machine stop cdap
+    $ docker exec -it cdap-sdk cdap-cli.sh
 
-     .. Windows
-     
-     > docker exec -d cdap-standalone /opt/cdap/sdk/bin/cdap sdk stop
-     > docker-machine stop cdap
+- Use the CDAP CLI in its own container (*cdap-cli*), against a remote CDAP instance at ``${CDAP_HOST}``::
+
+    $ docker run -it --name cdap-cli --rm caskdata/cdap-standalone cdap-cli.sh -u http://${CDAP_HOST}:11011
```

Comment:
`docker run -it --name cdap-cli --rm caskdata/cdap-standalone cdap cli -u http://${CDAP_HOST}:11011`